src/hooks/useHeaderCell.js

```diff
@@ -1,13 +1,16 @@
+import { useMemo } from 'react'
 import useManifest from './useManifest'
 
 export default (index) => {
   const { definition, setSorts, sorts, loading } = useManifest()
   const def = definition[index]
+  const columnSort = useMemo(() => sorts.find(s => s.id === def.id), [sorts, def])
```

Comment:
_while we are in here_ ... I would like to change the sort format from 
`[{id, isAsc<boolean>}, ...]` to `[{id, direction<string>}, ...]` where the direction code can be [ASCENDING, DESCENDING, NOT_SORTED]

We would need to export the direction codes on the main index too.